src/ctap/storage.rs

```diff
@@ -28,9 +28,11 @@ use alloc::vec;
 use alloc::vec::Vec;
 use arrayref::array_ref;
 use cbor::cbor_array_vec;
+use core::cmp;
 use core::convert::TryInto;
 use crypto::rng256::Rng256;
 use persistent_store::StoreUpdate;
+use persistent_store::fragment::{read_range, write};
```

Comment:
Since the names are generic, I had in mind to use them with the module name:

```rust
use persistent_store::{fragment, StoreUpdate};
// ...
fragment::read_range()
fragment::write()
```

As you want.

Reply:
Seems better, changed.